wiki/Tournaments/OWC/1/en.md

```diff
@@ -1,152 +1,165 @@
+| ![][flag_JP] Japan | **MeRcYyy**, taNa, Rigeru, Ozouni, Rorry, SiLviA |
+| ![][flag_KR] Korea | **KRZY**, Cookiezi, Reisen Udongein, ehRh, M A I D, Binjip |
+| ![][flag_MO] Macau | **Ballance**, wl01939929, starpun1, \_\_\_\_\_B, xinhe |
+| ![][flag_MY] Malaysia | **The 08 team\_Bourdon**, akupp, DragonSparta, Crimson_SoulZ, mycyber, mekadon |
+| ![][flag_NL] Netherlands | **GladiOol**, Henkie, Lesjuh, Awoken, Happy30, eddieee |
+| ![][flag_NZ] New Zealand | **NumoT123**, toejamms, Skye-, Acidsky-, Jiantz, Deadbeat |
+| ![][flag_PH] Philippines | **dayun10**, blacksymbian, jockeytiyan, wyndII, nachopiggy, bakaloidsky |
+| ![][flag_PL] Poland | **ShaggoN**, kuburaczek, White Wolf, Niko-, rEdo, fartownik |
+| ![][flag_PT] Portugal | **JonnyThatJonny**, creativ, sttailruby12, makkura |
+| ![][flag_RU] Russia | **GaShiK**, larch, Kotya, Akai-, CracK, TKiller |
+| ![][flag_ES] Spain | **X\_Ray**, Skullboss, hyperluigi, Lionheart69, choche, migul |
+| ![][flag_SE] Sweden | **beko1994**, Saten-san, palinus, Vikkez, Xgor, Darkoff |
+| ![][flag_TW] Taiwan | **Uan**, Rucker, 0222101916, Tomoka Rin, FuNnY--, NanaDesu |
+| ![][flag_UA] Ukraine | **gorlum**, rockleejkooo, Mosya, RainForce, gef |
+| ![][flag_GB] United Kingdom | **Doomsday**, Natteke, aevv, Jericho2442, DiamondCrash |
+| ![][flag_US] United States | **Lybydose**, Mafiamaster, Cyclone, ebacho, naptime |
+
+---------------------------------------------------
+
+![OWC #1 brackets](brackets.png "Brackets")
+
+----------------------------------------------------
+
+## Mappools
+
+### Finals
+
+| Order | Beatmap |
+|:-------:|---------|
+| 1 | [Glasslake - Driving Lazy Bee (DaRRi MIx)](https://osu.ppy.sh/s/3861) |
+| 2 | [YUI - Sea (Senior)](https://osu.ppy.sh/s/13021) |
+| 3 | [Suzaku - Anisakis -somatic mutation type "Forza" (Another)](https://osu.ppy.sh/s/15579) |
+
+**Break time**
+
+| Order | Beatmap |
+|:-------:|---------|
+| 4 | [IOSYS - Danzai Yamaxanadu (Eternal Damnation)](https://osu.ppy.sh/s/13654) |
+| 5 | [DJ YOSHITAKA - Bloody Tears(IIDX EDITION) (Insane)](https://osu.ppy.sh/b/29731) |
+| 6 | [Betwixt & Between - I think we can go to the moon (Another)](https://osu.ppy.sh/s/13728) |
+
+**Break time**
+
+| Order | Beatmap |
+|:-------:|---------|
+| Tiebreaker (3-3) | [Renard - Banned Forever (Nogard)](https://osu.ppy.sh/s/16349) |
+
+### Semifinals
+
+| Order | Beatmap |
+|:-------:|---------|
+| 1 | [COOL&CREATE - Rapid Ensemble (Lunatic)](https://osu.ppy.sh/s/13235) |
+| 2 | [Suzaku VS Genbu - Himiko (Another)](https://osu.ppy.sh/s/12710) |
+| 3 | [Susumu Hirasawa - Big Brother (KIRBY Mix)](https://osu.ppy.sh/s/10714) |
+| 4 | [NEWS - weeeek (Insane)](https://osu.ppy.sh/b/22422) |
+| 5 | [Tachibana Miya - Miya to Tengoku to Jigoku (Hentai)](https://osu.ppy.sh/s/17450) |
+| 6 | [Nobuo Uematsu - The Fierce Battle (Omega)](https://osu.ppy.sh/b/27466) |
+| Tiebreaker (3-3) | [Rhapsody - Emerald Sword (Extreme)](https://osu.ppy.sh/b/25580) |
+
+### Quarterfinals
+
+- NoMod
```

Comment:
All maps should have the creator of said map and link to their respective diff (/b/) link